src/libs/actions/User.js

```diff
@@ -285,10 +300,15 @@ function addPaypalMeAddress(address) {
 function deletePaypalMeAddress() {
     const optimisticData = [
         {
-            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            onyxMethod: CONST.ONYX.METHOD.SET,
             key: ONYXKEYS.NVP_PAYPAL_ME_ADDRESS,
             value: '',
         },
```

Comment:
Do we still need to do this if we have the ONYXKEYS.PAYPAL now?

Reply:
+1

Reply:
When we delete the paypal address, we still need to set paypal me address NVP to an empty string